mesos_slave/tests/test_integration_e2e.py

```diff
@@ -39,8 +39,13 @@ def assert_metrics_covered(aggregator):
     ):
         metrics.update(d)
 
+    expected_tags = ["instance:mytag1", "url:{}/metrics/snapshot".format(URL), "mesos_node:slave"]
+
     for _, v in iteritems(metrics):
         aggregator.assert_metric(v[0])
+        for tag in expected_tags:
+            aggregator.assert_metric_has_tag(v[0], tag)
```

Comment:
Why not `aggregator.assert_metric(v[0], tags=expected_tags)`

Reply:
Because the pid tag is dynamic so I think I need to test it separately.